packages-ts/gauntlet-terra-contracts/src/commands/abstract/batchWrapper.ts

```diff
@@ -0,0 +1,62 @@
+import AbstractCommand, { makeAbstractCommand } from '.'
+import { Result, WriteCommand } from '@chainlink/gauntlet-core'
+import { TerraCommand, TransactionResponse, logger } from '@chainlink/gauntlet-terra'
+import { AccAddress, LCDClient, Msg } from '@terra-money/terra.js'
+import { prompt } from '@chainlink/gauntlet-core/dist/utils'
+import { ExecutionContext } from './executionWrapper'
+import { parseJSON } from '@chainlink/gauntlet-terra/dist/lib/rdd'
+
+export const wrapCommand = (command) => {
+  return class BatchCommand extends TerraCommand {
+    static id = `${command.id}:batch`
+    subCommands: any[]
```

Comment:
```suggestion
    subCommands: TerraCommand[]
```

Reply:
So I made this of type `any` because the `AbstractCommand` `simulateExecute` function isn't available otherwise

Reply:
Use the `simulate` from TerraCommand instead. Also, the simulation should simulate the tx with every message included, this could reveal a tx size limit most probably. 
```
await this.simulate(this.wallet.key.accAddress, rawTx)
```